app/src/main/res/layout/child_comment_item.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
what's the reason for not use ConstraintLayout?

Reply:
I have no idea... I don't even remember doing this layout 😕 
I will redo it using constraint layout.